cmd/podman/search.go

```diff
@@ -2,18 +2,20 @@ package main
 
 import (
 	"context"
-	"reflect"
-	"strconv"
-	"strings"
-
+	"fmt"
 	"github.com/containers/image/docker"
+	"github.com/containers/image/types"
 	"github.com/pkg/errors"
 	"github.com/projectatomic/libpod/cmd/podman/formats"
 	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
 	"github.com/projectatomic/libpod/libpod/common"
 	sysreg "github.com/projectatomic/libpod/pkg/registries"
+	"github.com/projectatomic/libpod/pkg/util"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
+	"reflect"
```

Comment:
The builtin imports should be at the top separated by a while line from the github imports.

Reply:
I'm a little surprised lint didn't complain about this.